File: 백준/Silver/13241. 최소공배수/최소공배수.java
import java.io.BufferedReader;
import java.io.IOException;
import java.io.InputStreamReader;

public class Main {
    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        String[] input = br.readLine().split(" ");
        long a = Long.parseLong(input[0]);
        long b = Long.parseLong(input[1]);
        System.out.println((a*b) / gcd(a, b));
    }

    private static long gcd(long a, long b) {
        if (a % b == 0) {
            return b;
        }
        return gcd(b, a%b);
    }
}

File: 백준/Bronze/1934. 최소공배수/최소공배수.py
import sys

sys.setrecursionlimit(100000)

def gcd(a, b):
    if a % b == 0:
        return b
    return gcd(b, a % b)


t = int(input())
for _ in range(t):
    n, m = map(int, input().split(" "))
    print((n* m) // gcd(n, m))